libminifi/include/core/state/nodes/AgentInformation.h

```diff
@@ -643,16 +704,7 @@ class AgentInformation : public DeviceInformation, public AgentMonitor, public A
   }
 
   std::vector<SerializedResponseNode> serialize() {
```

Comment:
Can the `serialize()` member functions be marked as `const`? They don't seem to mutate the object.